### PR TITLE
chore: use shared rollup config

### PR DESCRIPTION
**Pull request by jeswr**

### CODE REVIEW

File: rollup.config.mjs [hunk 1]
```diff
@@ -1,48 +1,40 @@
...
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+import createConfig from "@inrupt/base-rollup-config";
+import pkg from "./package.json" assert { type: "json" };
 
-// Until we only support Node 18+, this should be used instead
-// (see https://rollupjs.org/guide/en/#importing-packagejson) 
-import { createRequire } from 'node:module';
-const require = createRequire(import.meta.url);
-const pkg = require('./package.json');
```
Comment (NSeydoux) [inline rollup.config.mjs:8]: We haven't officially removed support for Node 16 yet.

**Review by NSeydoux — CHANGES_REQUESTED** [review]

File: rollup.config.mjs (hunk 1 above)
Comment (jeswr) [reply rollup.config.mjs:8]: Yes - but we only need node 18 to run the build step. This doesn't affect whether our packaged library supports node 16.

Comment (NSeydoux) [reply rollup.config.mjs:8]: That is correct, but the CI runs the build step on all supported versions. We should consider building the code once, and then running the tests in the different target runtime, but our code isn't structured to support this so far.

Comment (jeswr) [reply rollup.config.mjs:8]: It actually appears to be running on node 16 fine (https://github.com/inrupt/solid-client-vc-js/actions/runs/6856123821/job/18642653808?pr=863); looks like it was backported to 16.15.0 (https://stackoverflow.com/questions/73824694/node-16-is-the-experimental-json-modules-cli-flag-still-required).

Comment (NSeydoux) [reply rollup.config.mjs:8]: Ah ok, I didn't realize that, no problem then.

**Review by NSeydoux — APPROVED** [review]

**Pull request merged**